drive/sheets.py

```diff
@@ -12,8 +12,8 @@
 
 
 def _auto_retry(fn: Callable[[], T], *,
-                max_retries=2,
-                sleep_for=4) -> T:
+                max_retries: int = 2,
```

Comment:
```suggestion
                max_retries = 2,
```
`2` is already an `int`